DOCSUP-8711: Document the sumCount aggregate function and optimize_fuse_sum_count_avg setting (#24459)
ClickHouse/ClickHouse: docs/en/sql-reference/aggregate-functions/reference/sumcount.md

---
toc_priority: 142
---

# sumCount {#agg_function-sumCount}

Calculates the sum of the numbers and counts the number of rows at the same time.

**Syntax**

``` sql
sumCount(x)
```

**Arguments** 

-   `x` — Input value, must be [Integer](../../../sql-reference/data-types/int-uint.md), [Float](../../../sql-reference/data-types/float.md), or [Decimal](../../../sql-reference/data-types/decimal.md).

**Returned value**

> -   Tuple `(sum, count)`, where `sum` is the sum of numbers and `count` is a counts the number of rows or not-NULL values.

Reviewer: ```suggestion
-   Tuple `(sum, count)`, where `sum` is the sum of numbers and `count` is the number of rows with not-NULL values.
```
